dev/com.ibm.ws.microprofile.opentracing.jaeger.adapter/resources/com.ibm.ws.microprofile.opentracing.jaeger.resources/JaegerMessages.nlsprops

```diff
@@ -0,0 +1,84 @@
+#
+# {0,date} is a code that tells java to insert a parameter into
+# this space when displaying it to the user.  In this case,
+# the parameter that is inserted is a date.
+#
+# The 0 indicates that this is where the first parameter is
+# inserted in the string.  There may be more than one parameter
+# in any one string.  The 'date' tells java to format this
+# parameter as a date.
+#
+# The {0,date} code should be placed at the appropriate position
+# in the translated text.  Please note that the code in the
+# translation must match exactly the code in the original
+# string.
+#
+# The developer should describe in comments above the string to be
+# translated each of the parameters so that you will be able to
+# determine where the best location for each parameter in the
+# translated string.
+#
+# Lines that begin with a pound sign (#) are comments and should not be
+# translated.
+# -----------------------------------------------------------------------
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit: 
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+#
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by Opentracing Impl -  range is 0000-4999, plus 9999 for the emergency message
+# This bundle uses range 1000-1100
+#-----------------------------------------------------------------------------------------------------------------------------
+JAEGER_TRACER_CREATED=CWMOT1001I: An instance of JaegerTracer has been created for application {0}.  Tracing information is sent to {1}.
+JAEGER_TRACER_CREATED.explanation=An instance of JaegerTracer has been created to send spans from the application to the tracing backend.
+JAEGER_TRACER_CREATED.useraction=No action required.
+
+JAEGER_ENV_VAR_PARSE_ERROR=CWMOT1002W: The value of system property or environment variable {0} is not a valid type of {1}.
+JAEGER_ENV_VAR_PARSE_ERROR.explanation=The value defined in the system property or environment variable cannot be converted to the type.
+JAEGER_ENV_VAR_PARSE_ERROR.useraction=Define a value for the system property or environment variable that matches the type. 
+
+JAEGER_PASSWORD_CANNOT_DECODE=CWMOT1003W: The value of JAEGER_PASSWORD cannot be decoded.
+JAEGER_PASSWORD_CANNOT_DECODE.explanation=The password cannot be decoded and therefore it is ignored.
+JAEGER_PASSWORD_CANNOT_DECODE.useraction=Use securityUtility to encode passwords.
+
+JAEGER_TAGS_CANNOT_PARSE=CWMOT1004W: The value of JAEGER_TAGS cannot be parsed.
+JAEGER_TAGS_CANNOT_PARSE.explanation=The tags cannot be parsed and therefore they are ignored.
+JAEGER_TAGS_CANNOT_PARSE.useraction=Use a comma separated name=value pairs.
+
+JAEGER_CONFIG_EXCEPTION=CWMOT1005E: JaegerTracer cannot be instantiated.  Reason: {0} 
+JAEGER_CONFIG_EXCEPTION.explanation=JaegerTracer cannot be instantiated due to configuration issues.
```

Comment:
Change
"JaegerTracer cannot be instantiated due to configuration issues."
To
"A JaegerTracer object cannot be instantiated due to configuration issues."